lib/winrm/wsmv/init_runspace_pool.rb

```diff
@@ -35,7 +35,10 @@ def create_header(header)
       end
 
       def create_body(body)
-        body.tag!("#{NS_WIN_SHELL}:Shell", 'ShellId' => shell_id) { |s| s << Gyoku.xml(shell_body) }
+        # OMI server requires a Name for the Shell
+        body.tag!("#{NS_WIN_SHELL}:Shell", 'ShellId' => shell_id, 'Name' => 'Runspace') do |s|
```

Comment:
I'm not sure if there's any reason to make this name unique or not...

Reply:
Looks like no:

```
An optional xs:string value that, if present, MUST contain a user-friendly name for the shell. There are no uniqueness requirements for this value.
```
According to https://msdn.microsoft.com/en-us/library/cc251546.aspx